config/ext/rails.yml

```diff
@@ -589,6 +661,12 @@ Rails/RequestReferer:
     - referer
     - referrer
 
+Rails/RequireDependency:
+  Description: 'Do not use `require_dependency` when running in Zeitwerk mode. `require_dependency` is for autoloading in classic mode.'
+  Reference: 'https://guides.rubyonrails.org/autoloading_and_reloading_constants.html'
+  Enabled: false
```

Comment:
You might want to add a comment, that we enable this feature in a later stage. Currently we do not have all applications running with Zeitwerk.